src/NServiceBus.Core.Tests/Persistence/InMemory/InMemoryOutboxPersisterTests.cs

```diff
@@ -19,7 +19,14 @@ class InMemoryOutboxPersisterTests
             var messageId = "myId";
 
             var messageToStore = new OutboxMessage(messageId, new[] { new TransportOperation("x", null, null, null) });
-            await storage.Store(messageToStore, null, new ContextBag());
+            // TODO: It is not clear to me why I need to await a transaction?
```

Comment:
Would it be likely that storages will need to do IO in order to "prepare" a transaction?